server_addon/deadline/client/ayon_deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -459,7 +460,8 @@ def _tile_render(self, payload):
         frame_tile_job_id = {}
         for frame, tile_job_payload in frame_payloads.items():
             job_id = self.submit(tile_job_payload,
-                                 instance.data["deadline"]["auth"])
+                                 instance.data["deadline"]["auth"],
```

Comment:
don't you have `auth` and `verify` variables now?
```python
job_id = self.submit(tile_job_payload, auth, verify)
```

Reply:
Ohh, I do. Updated.